### ISSUE TITLE
Updating the memory allocation

### ISSUE THREAD

**Issue by tixtix** [jlesage/docker-crashplan-pro#52]
I'm fairly new to Linux commands and Docker setup.

I have a successfully running backup but I'm running low on a memory. Can anyone share a command to set the memory allocation for an already running backup? I have 16GB RAM available on my Synology and my backup is over 12TB.

Where do I enter the command "CRASHPLAN_SRV_MAX_MEM" ?

Thanks!

**Comment by excalibr18** [comment]
It goes in the "docker run" command when you first set up the container. For example:

docker run -d --name=crashplan-pro -e USER_ID=0 -e GROUP_ID=0 -e CRASHPLAN_SRV_MAX_MEM=8192M -p 5800:5800 -v /volume1/docker/appdata/crashplan-pro:/config:rw -v /volume1/:/volume1:ro jlesage/crashplan-pro

This will allocate 8 gigs of memory to it.

**Comment by jlesage** [comment]
See https://github.com/jlesage/docker-crashplan-pro/blob/master/README.md#crashes--maximum-amount-of-allocated-memory for more details.

**Comment by tixtix** [comment]
@excalibr18

Thanks for the help!

As you pointed it out, this command is used when setting up the container for the first time. How do I change the memory allocation now? 

If I run this command "docker run -d --name=crashplan-pro -e USER_ID=0 -e GROUP_ID=0 -e CRASHPLAN_SRV_MAX_MEM=8192M -p 5800:5800 -v /volume1/docker/appdata/crashplan-pro:/config:rw -v /volume1/:/volume1:ro jlesage/crashplan-pro" will it affect somehow my current upload?

Basically, I'm looking for a command to change the memory allocation, on a running backup.

**Comment by tixtix** [comment]
@jlesage Thanks for the links but there is no mention on how to change the memory allocation, on an already running backup. I see the command and the parameters but I need a noobie guidance ;)

FYI, I do not have a running CrashPlan/Code42 desktop app as it refuses to connect to my backup on Synology...

**Comment by excalibr18** [comment]
@tixtix: You'll have to stop the container, delete the container, then create a new one.  You shouldn't need to re-back up everything.  CrashPlan will scan and compare what's already been upload and then pick up from where it left off.

**Comment by tixtix** [comment]
@excalibr18 I understand. This is what I was trying to avoid hoping that there is a command only for changing the memory allocation. I guess there is not. Thanks!

If there is one more thing, if I can bother you? 
How can I make a CrashPlan/Code42 desktop app running? Back in the "Patter" days, I had to edit the .ui_info file. Somehow it doesn't work this time around. Any ideas? 
Thanks!

**Comment by excalibr18** [comment]
You're welcome!

**Comment by jlesage** [comment]
@tixtix, I will add to the README some details about how to change parameters of a running container.

I guess this issue can now be closed.
